docs/source/campaigns.rst

```diff
@@ -17,5 +17,5 @@ High-resolution analysis (``campaign = "high_res_v1"``) is only appropriate for
 (e.g., those with the oRRS18to6 ocean and sea-ice mesh).
 It will only work if you:
 
-- have access to the `bigmem nodes on Cori <https://docs.nersc.gov/systems/cori-largemem/#access-to-the-large-memory-nodes>`_
+- have access to the `bigmem nodes on NERSC Cori <https://docs.nersc.gov/systems/cori-largemem/#access-to-the-large-memory-nodes>`_
```

Comment:
Is there an equivalent for Perlmutter?

Reply:
Nothing is currently listed in NERSC's documentation about large-memory nodes for Perlmutter.

Reply:
So does that mean the high-res campaign won't work now? Since it apparently only runs on Cori's `bigmem` nodes?